js/src/components/AnalysisUpsell.js

```diff
@@ -61,7 +66,9 @@ const AnalysisUpsell = ( props ) => {
 						__( "Go %s!", "wordpress-seo" ),
 						"Premium"
 					) }
-					<SvgIcon icon="arrow-right" size="8px" color={ colors.$color_black } />
+					<Caret>
```

Comment:
The link is a flex container, so the text and svg icon were both flex items. This was done to align them vertically with `align-items: center`. Now, the new div is a flex item while the svg icon isn't any longer. However, it still uses some flex CSS properties. Instead of adding a new div, I'd recommend to just style the SVG icon, e.g.:
```
const StyeldSvgIcon = styled( SvgIcon )`
	margin-left: 3px;
`;
```

Reply:
Note: for RTL languages this should be reversed using `getRtlStyle()` anyways.